source/frontend/src/components/maps/leaflet/Markers/SingleMarker.tsx

```diff
@@ -60,23 +64,23 @@ const SinglePropertyMarker: React.FC<React.PropsWithChildren<SinglePropertyMarke
         latlng: latlng,
         pimsLocationFeature: pointFeature.properties,
         pimsBoundaryFeature: null,
-        pmbcFeature: null,
+        fullyAttributedFeature: null,
       });
     } else if (isPimsBoundary(pointFeature)) {
       mapMachine.mapMarkerClick({
         clusterId: clusterId,
         latlng: latlng,
         pimsLocationFeature: null,
         pimsBoundaryFeature: pointFeature.properties,
-        pmbcFeature: null,
+        fullyAttributedFeature: null,
       });
-    } else if (isParcelMap(pointFeature)) {
+    } else if (isFaParcelMap(pointFeature)) {
       mapMachine.mapMarkerClick({
         clusterId: clusterId,
         latlng: latlng,
         pimsLocationFeature: null,
         pimsBoundaryFeature: null,
-        pmbcFeature: pointFeature.properties,
+        fullyAttributedFeature: pointFeature.properties,
```

Comment:
This code block will never execute on local development - I'd like to know if that would impact what we can click on the map while doing local development